addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/handler/HarmonyHubHandler.java

```diff
@@ -209,7 +216,26 @@ private synchronized void connect() {
         discoTime = config.discoveryTimeout > 0 ? config.discoveryTimeout : DISCO_TIME;
         heartBeatInterval = config.heartBeatInterval > 0 ? config.heartBeatInterval : HEARTBEAT_INTERVAL;
 
-        discovery = new HarmonyHubDiscovery(discoTime);
+        String host = config.host;
+
+        // earlier versions required a name and used network discovery to find the hub and retrieve the host,
```

Comment:
Right, but the only way to get the hub session id is to start the discovery process, we have to send the HUB a UDP messages and then wait for it to send us back a message containing the login info.  This is 99% the discovery logic, so I'm not sure how to avoid this here.